stock_indicators/_cslib/__init__.py

```diff
@@ -20,9 +20,12 @@
 
 from System import DateTime as CsDateTime
 from System import Decimal as CsDecimal
+from System import Enum as CsEnum
 from System.Globalization import CultureInfo
 from System.Collections.Generic import List as CsList
 
 from Skender.Stock.Indicators import Indicator as CsIndicator
 from Skender.Stock.Indicators import Quote as CsQuote
 from Skender.Stock.Indicators import ResultBase as CsResultBase
+from Skender.Stock.Indicators import CandlePart as CsCandlePart
```

Comment:
Did you figure out how to use enum in Python, from the library?

Reply:
This usage is available because the Enum class is just used as result values only in Python-side, not a parameter for dll function.

Reply:
The reason why we can't use Enum is caused when we take Enum as a parameter to C# function.